packages/orchestrator-ui-components/src/components/WfoKeyValueTable/styles.ts

```diff
@@ -44,6 +44,8 @@ export const getStyles = ({ theme }: WfoTheme) => {
     const valueColumnStyle = css({
         borderTopRightRadius: radius,
         borderBottomRightRadius: radius,
+        paddingTop: theme.base / 2,
+        paddingBottom: theme.base / 2,
```

Comment:
I think it might be better to use `theme.size.s` ( which is also 8px ), because `/2` could result in half pixels if `theme.base` is ever changed in the future.

Reply:
done